config.go

```diff
@@ -18,7 +18,7 @@ type Configs struct {
 	AppPath                 string          `env:"app_path,required"`
 	ExpansionfilePath       string          `env:"expansionfile_path"`
 	Track                   string          `env:"track,required"`
-	UserFraction            string          `env:"user_fraction,opt[0.05,0.1,0.2,0.5]"`
+	UserFraction            string          `env:"user_fraction,range]0.0..1.0["`
```

Comment:
could you please pass the api description of this range?

Reply:
Sure:
https://developers.google.com/android-publisher/api-ref/edits/tracks
See releases[].userFraction